tests/README.md

```diff
@@ -117,6 +133,10 @@ molecule destroy -s centos-8
 
 See [Running the tests](#running-the-tests) section for more information on available options.
 
+### Skipping specific tests or modules
+
```

Comment:
It seems that there is text missing here.

Reply:
Thank you. Fixed.